File: PersonnelManagement/PersonnelManagement/Front/classes/personcontract.ts
import Personadditionalagreement from './personadditionalagreement';

export default class Personcontract {
    id: number;
    person: number;
    pay: number;
    datestart: Date;
    dateend: Date;
    ordernumber: string;
    ordernumbertype: string;
    orderdate: Date;
    orderwho: string;
    orderwhoid: number;
    orderid: number;
    sourceoffinancing: number;
    payvalue: number;
    stateserviceyears: number;
    stateservicemonths: number;
    stateservicedays: number;
    vacationdays: number;
    personadditionalagreements: Personadditionalagreement[];

    datestartString: string;
    dateendString: string;
    payBool: boolean;
    orderdateString: string;
}

File: PersonnelManagement/PersonnelManagement/Models/Personcontract.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Personcontract
    {
        public int Id { get; set; }
        public int Person { get; set; }
        public DateTime Datestart { get; set; }
        public DateTime Dateend { get; set; }
        public sbyte Pay { get; set; }
        public string Ordernumber { get; set; }
        public string Ordernumbertype { get; set; }
        public DateTime? Orderdate { get; set; }
        public string Orderwho { get; set; }
        public int Orderwhoid { get; set; }
        public int Orderid { get; set; }
        public int Sourceoffinancing { get; set; }
        public int Payvalue { get; set; }
        public int Stateserviceyears { get; set; }
        public int Stateservicemonths { get; set; }
        public int Stateservicedays { get; set; }
        public int Vacationdays { get; set; }
    }
}
